build.sh

```diff
@@ -383,7 +381,6 @@ if [ ${BUILD_BE} -eq 1 ] ; then
                   -DWITH_CLANG_TIDY=${WITH_CLANG_TIDY}                  \
                   -DWITH_COMPRESS=${WITH_COMPRESS}                      \
                   -DWITH_STARCACHE=${WITH_STARCACHE}                    \
-                  -DWITH_BRPC_KEEPALIVE=${WITH_BRPC_KEEPALIVE}          \
```

Comment:
Why to remove the `WITH_BRPC_KEEPALIVE`?

Reply:
@stdpain

Reply:
Before brpc 1.8 this parameter did not exist so he was needed to control it. Now that we have upgraded to brpc 1.9, this variable is always present. We can turn it on by default and control it by config.